src/main/glc2lndMod.F90

```diff
@@ -293,6 +299,23 @@ subroutine set_glc2lnd_fields(this, bounds, glc_present, x2l, &
        end if
     end if
 
+    if (glc_do_dynglacier) then
```

Comment:
You put this in a routine that is called every time in the run loop. Can you move this to initialization so that it is only done once?